solutions/cses-1619.mdx

```diff
@@ -184,6 +188,72 @@ int main(){
 }
 ```
 
+</CPPSection>
+	
+<JavaSection>
+	
+<Warning>
+Java solutions using this implementation will TLE on test case #6.
+</Warning>
+
+```java
+import java.io.*;
+import java.util.*;
+
+public class RestaurantCustomers {
+	static class Customer {
+		int time;
+		// a type of 1 means that it is an arrival
+		// and a type of 2 means that it is a departure
+		int type;
+	}
+
+	static class CustomerComparator implements Comparator<Customer> {
+		public int compare (Customer one, Customer two) {
+			// use Integer.compare to avoid writing if statements
+			return Integer.compare(one.time, two.time);
+		}
+	}
+
+	public static void main(String[] args) {
+		Kattio io = new Kattio();
+		int n = io.nextInt();
+		Customer[] data = new Customer[2 * n];
+
+		for (int x = 0; x < n; x++) {
+			Customer c1 = new Customer();
+			c1.time = io.nextInt();
+			c1.type = 1;
+
+			Customer c2 = new Customer();
+			c2.time = io.nextInt();
+			c2.type = -1;
+
+			data[2 * x] = c1;
+			data[2 * x + 1] = c2;
+		}
+
+		Arrays.sort(data, new CustomerComparator());
+
+		int sum = 0;
+		int answer = 0;
+		for (Customer customer : data) {
+			// if it is an arrival, add 1
+			// if it is a departure, subtract 1
+			sum += customer.type;
+			answer = Math.max(answer, sum);
+		}
+		io.println(answer);
+		io.close();
+	}
+	//CodeSnip{Kattio}
+}	
+```
+	
```

Comment:
```suggestion
```

Reply:
actually i think that will mess up the display

Reply:
👍